helm/README.md

```diff
@@ -0,0 +1,64 @@
+# source{d} blog deployment
+
+This is the [Helm](https://helm.sh/) chart to deploy [source{d} blog](https://blog.sourced.tech) in kubernetes.
+
+# Introduction
+
+The blog website is generated using [Hugo](http://gohugo.io/)
+and served with [Caddy](https://caddyserver.com/) web server.
+A docker container is built by our self hosted [Drone](https://drone2.srcd) server,
+and deployed using this [helm plugin](https://github.com/ipedrazas/drone-helm) in our GKE kubernetes clusters.
+
+# Workflow:
+
+Pushes to master branch in the [repository](https://github.com/src-d/blog), trigger a drone job.
+A new staging docker image is built and uploaded to [Quay registry](https://quay.io/repository/srcd/blog).
+It is labeled with the commit hashtag.
+The blog pods running on our k8s _staging_ cluster are then upgraded to use this new image.
+
+If a tag in the form is pushed along with the master branch HEAD,
+then a new production docker image is built too,
+and uploaded and labeled in an analog way.
+Then the pods running on our k8s _production_ cluster are upgraded to use this new image.
+
```

Comment:
This Workflow is not related to the chart. This information should go somewhere else.